dispatcher/frontend-ui/src/components/PipelineTable.vue

```diff
@@ -180,12 +180,20 @@
       },
       async loadLastRuns() {
         let parent = this;
+        parent.last_runs_loaded = false;
         let schedule_names = parent.tasks.map(function (item) { return item.schedule_name; }).unique();
 
         parent.toggleLoader("fetching last runs…");
-        let requests = schedule_names.map(function (schedule_name) {
-            return parent.queryAPI('get', "/schedules/" + schedule_name);
-          });
+        const chunkSize = 5;
+        let requests = []
+        for (let i = 0; i < schedule_names.length; i += chunkSize) {
+            const chunk = schedule_names.slice(i, i + chunkSize);
+            for (let iChunk in chunk) {
+              requests.push(parent.queryAPI('get', "/schedules/" + chunk[iChunk]))
+            }
+            await Constants.delay(500)
```

Comment:
Is this 500milliseconds? Would that mean that it would spread the 200 requests over 20seconds ?
200 / 5 = 40 iterations
40 * 500ms = 20Kms = 20s
That seems crazy long

Reply:
Just as crazy as making 200 requests for displaying one screen. What would you suggest?

Reply:
If the value can't obviously set, it should be changeable with an env. I'd start with 100 and we'll see how it goes.

Reply:
Making the nb of schedules in chunk changeable would help as well

Reply:
Done